coral-component-calendar/src/scripts/Calendar.js

```diff
@@ -613,11 +617,8 @@ class Calendar extends BaseFormField(BaseComponent(HTMLElement)) {
     
     if (el) {
       if (isTransitioning) {
-        commons.transitionEnd(newTable, () => {
-          // Prevents from chopping off the transition animation
-          window.setTimeout(() => {
-            el.querySelector('._coral-Calendar-date').classList.add('is-focused');
-          }, 50);
+        window.requestAnimationFrame(() => {
```

Comment:
So the animation isn't chopped off neither with `requestAnimationFrame` ?

Reply:
I'm not noticing animation getting chopped off, although I'm not quite sure what is meant by that, and the movement of focus is much faster without multiple dates getting highlighted when moving quickly between months.